### ISSUE TITLE
[XC / XPM] SPS 1.8.1 correção em datas de publicação <pub-date>

### ISSUE THREAD

**Issue by RPostalli** [scieloorg/PC-Programs#3065]
Relação com scieloorg/scielo_publishing_schema#811

Para não quebrar compatibilidade documentar apenas a inclusão de pub-date do tipo collection e forçar o uso de pub-date do tipo epub sempre com datas completas (dia +mês+ano). Retirar suporte de epub-ppub.

Exemplo:

```
<pub-date pub-type="epub">
	<day>01</day>
        <month>02</month>
        <year>2014</year>
    </pub-date>
<pub-date pub-type="collection">
        <season>Jan-Feb</season>
        <year>2014</year>
</pub-date>

```

Collection pode ser:
mês + ano
season + ano
ano

Epub **tem** que ser:
dia + mês + ano

**Comment by robertatakenaka** [comment]
@RPostalli : incluir ou reforçar que o único tipo que não tem collection é o aop.

**Comment by RPostalli** [comment]
@robertatakenaka sim, na documentação SPS farei bem detalhado como devem ser as datas.

**Comment by cinthia-vieira** [comment]
Não mostra erro quando:

- há ausência de `"collection"`;
- data `"collection"` com tag `<day>`;
- uso de `"epub-ppub"`;
- quando utilizadas quaisquer datas da versão 1.8 (`"epub-ppub`"; `"epub"` sem `"collection"`);
- uso do atributo `@date-type` (se a combinação de valores para o atributo for `"epub + collection`)
